cmd/crc/cmd/config/config.go

```diff
@@ -94,7 +94,7 @@ func less(lhsKey, rhsKey string) bool {
 	return lhsKey < rhsKey
 }
 
-func configurableFields(config *config.Config) string {
+func configurableFields(config config.Storage) string {
```

Comment:
Actually, `configurableFields` is more about `config.Schema` than `config.Storage`. But not really convenient to pass a `config.Schema` around, so I'd keep `config *config.Config`